paper/main.tex

```diff
@@ -415,22 +414,22 @@ \subsubsection*{Recovering dynamic higher-order correlations}
 \subsection*{Cognitively relevant dynamic high-order correlations in
   fMRI data}
 We used across-participant temporal decoders to identify cognitively
-relevant neural patterns in fMRI data (see \textit{Forward inference
-  and decoding accuracy}).  The dataset we examined~\citep[collected
+relevant neural patterns in fMRI data (see Forward inference
+  and decoding accuracy).  The dataset we examined~\citep[collected
 by][]{SimoEtal16} comprised four experimental conditions that exposed
 participants to stimuli that varied systematically in how cognitively
-engaging they were.  The \textit{intact} experimental condition had
+engaging they were.  The intact experimental condition had
 participants listen to an audio recording of a 10-minute story.  The
-\textit{paragraph}-scrambled experimental condition had participants
+paragraph-scrambled experimental condition had participants
```

Comment:
change to:

"paragraph-scrambled experimental condition (paragraph) had participants..."

Reply:
done